### PR TITLE
Arrow functions revisited

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment/commit if anything is wrong

### CODE REVIEW

File: 1-js/06-advanced-functions/12-arrow-functions/article.md [hunk 1]
```diff
@@ -112,15 +112,15 @@ function defer(f, ms) {
...
-- Do not have `arguments`
-- Can't be called with `new`
-- They also don't have `super`, but we didn't study it yet. We will on the chapter <info:class-inheritance>
+- دارای `this` نیستند
+- دارای `arguments` نیستند
+- نمی‌توانند همراه با `new` فراخوانی شوند
+- همچنین آن‌ها `super` ندارند اما هنوز آن را نخوانده‌ایم. در فصل <info:class-inheritance> آن را خواهیم خواند
 
-That's because they are meant for short pieces of code that do not have their own "context", but rather work in the current one. And they really shine in that use case.
+به دلیل اینکه آن‌ها برای قطعه‌های کوتاهی از کد در نظر گرفته شده‌اند که «زمینه» خودشان را ندارند و در زمینه کنونی کار می‌کنند. و آن‌ها واقعا در این مورد استفاده می‌درخشند.
```
Comment (mahdiHashemi14) [inline 1-js/06-advanced-functions/12-arrow-functions/article.md:126]: This line needs attention

Comment (mahdyar) [reply 1-js/06-advanced-functions/12-arrow-functions/article.md:126]:
```suggestion
به دلیل اینکه آن‌ها برای قطعه‌های کوتاهی از کد در نظر گرفته شده‌اند که «موقعیت»(context) خودشان را ندارند و در موقعیت کنونی کار می‌کنند. و آن‌ها واقعا در این مورد استفاده کاربرد دارند.
```

File: 1-js/06-advanced-functions/12-arrow-functions/article.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
-# Arrow functions revisited
+# بازدید دوباره از تابع‌های کمانی
```
Comment (mahdyar) [inline 1-js/06-advanced-functions/12-arrow-functions/article.md:1]: I guess "سرکشی" would be a better translation for visit. something like "سرکشی دوباره".

**Review by mahdyar — CHANGES_REQUESTED** [review]

File: 1-js/06-advanced-functions/12-arrow-functions/article.md (hunk 1 above)
Comment (mahdiHashemi14) [reply 1-js/06-advanced-functions/12-arrow-functions/article.md:126]: I translated "context" as "زمینه" in other articles. I think "موقعیت" is a good translation for "position" or "location" or something like this. I'll add the English term in parentheses.

**Comment by mahdiHashemi14** [comment]
Thank you @mahdyar 
/done

**Review by mahdyar — APPROVED** [review]
LGTM;
Thank you, @mahdiHashemi14!

**Pull request merged**